taxi/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from .models import Driver, Car, Manufacturer
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    fieldsets = (
```

Comment:
Do not redefine those attributes
Instead just modify them
E.g. list_display = UserAdmin.list_display + ("license_number",)